website/landing/styles/breakpoints.ts

```diff
@@ -0,0 +1,15 @@
+export type Breakpoint = "sm" | "md" | "lg" | "xg";
```

Comment:
@smhutch what if we publish the design tokens to keep all applications up to date with the latest changes?